.github/workflows/test-mobile.yml

```diff
@@ -1,45 +1,143 @@
 name: "[Test] Mobile App"
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-      - release
-      - hotfix
+  # push:
+  #   branches:
+  #     - main
+  #     - develop
+  #     - release
+  #     - hotfix
+  schedule:
+    - cron: "0 2/12 * * *"
   pull_request:
   workflow_dispatch:
+    inputs:
+      ref:
+        description: "The branch to run tests on (develop by default)"
+        required: false
+        default: develop
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: ${{ github.ref_name != 'develop' }}
 
-jobs:
-  codecheck:
-    name: "Ledger Live Mobile CodeCheck"
+jobs:  
+  detox-tests-ios:
+    name: "Ledger Live Mobile - iOS Detox Tests"
+    runs-on: macos-latest
     env:
       NODE_OPTIONS: "--max-old-space-size=7168"
-      FORCE_COLOR: 3
-
-    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: pnpm/action-setup@v2
+        with:
+          ref: develop # When tests are run on every push to PR: ${{ github.event.pull_request.head.sha || github.event.push.head.sha }}
```

Comment:
since we are using workflow dispatch we could use something like `github.event.input.ref || 'develop'` (again, it's just an example we probably did something like that in other jobs i think)

Reply:
Cool, it was `github.event.inputs.ref`